test/test.cpp

```diff
@@ -15,7 +15,7 @@ TEST_CASE("successful compress - pointer") {
     const char * pointer = data.data();
 
     std::string value = gzip::compress(pointer, data.size());
-    REQUIRE(value.size() > 0);
+    REQUIRE(!value.empty());
 }
```

Comment:
That is nice that `clang-tidy` changed this. `value.empty()` is slightly faster than `value.size() > 0`. While that does not matter in tests, good idea to use elsewhere.